bzr.go

```diff
@@ -64,6 +65,12 @@ func (s BzrRepo) Vcs() Type {
 
 // Get is used to perform an initial clone of a repository.
 func (s *BzrRepo) Get() error {
+
+	basePath := path.Join(s.LocalPath(), "..")
+	if _, err := os.Stat(basePath); err != nil {
```

Comment:
The error should be checked to make sure it's a not exists error which can be done with [`os.IsNotExist(err)`](https://golang.org/pkg/os/#IsNotExist) instead or the `err != nil` section. Any other error should return an error.